tests/phpunit/class-wc-stripe-test.php

```
@@ -1000,6 +1000,38 @@ public function test_collaborators_not_reinstantiated_on_second_construction():
+				has_action( $hook, [ $instance, $callback ] ),
+				"{$label}: constructor must not register {$hook}"
+			);
+
+			$instance->register_hooks();
+
+			$this->assertNotFalse(
+				has_action( $hook, [ $instance, $callback ] ),
+				"{$label}: register_hooks() must register {$hook}"
+			);
+		}
+	}
```

PHPMD (2.15.0)

[error] The method test_core_collaborator_constructors_do_not_register_hooks is not named in camelCase. (undefined)

(CamelCaseMethodName)